<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="saveStatusButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>500</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>保存状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoSolveButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>340</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>自动求解</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rankBrowse">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>420</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>排名记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameSettingButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>580</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>180</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoStepButton">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>260</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitGameButton">
    <property name="geometry">
     <rect>
      <x>1085</x>
      <y>140</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton0">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton1">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>200</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>345</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton4">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>345</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>345</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton7">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>490</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton8">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="picButton6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>135</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameScoreLCD">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正喵呜体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="picturePreview">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>200</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameTimeLCD">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>485</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正喵呜体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gameStepLCD">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正喵呜体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
